src/utils.cjs

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  USE_SOURCE_MAPS: process.env.USE_SOURCE_MAPS === 'true',
+  replacePathWithSourcePath: path => path.replace(/\/dist\//, '/src/').replace('.js', '.ts'),
```

Comment:
I guess if this wanted to be merged upstream this (`/dist` and `/src`) should be configurable.